DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -570,15 +586,27 @@ static function bool BingoGoalImpossible(string bingo_event, int start_map, int
         break;
 
     case 6: // Hong Kong
-    case 7:// fallthrough to 2nd half of Hong Kong
+        // switch(bingo_event)
+        // {
+        // // these two goals can actually be done with the way these starts currently work, but would normally be impossible
+        // case "ClubEntryPaid":
+        // case "M06JCHasDate":
+        //     return start_map > 65;
+        // }
+    case 7: // fallthrough to 2nd half of Hong Kong
         switch(bingo_event)
         {
+        case "MaggieCanFly":
+        case "PoliceVaultBingo": // unless and until a datacube with the code gets added
```

Comment:
it's good to mark comments in a way that makes them easy to search for and find later, for this we use the ScummVM convention of `TODO:` or `HACK:`

```suggestion
        case "PoliceVaultBingo": // TODO: unless and until a datacube with the code gets added
```